server/controllers/activitypub/lib/radio_hooks.ts

```diff
@@ -42,11 +42,12 @@ const createDebouncedActivity = ({ userId, shelfId }: createActivityParams) => a
 }
 
 async function _createDebouncedActivity ({ userId, shelfId }: createActivityParams) {
-  let name, user
+  let name, user, poolActivities
   if (userId) {
     delete debouncedActivities[userId]
     user = await getUserById(userId) as User
     if (!user.fediversable) return
+    ({ poolActivities } = user)
```

Comment:
[non-blocking]
Please prefix lines starting with `(` with a `;`, I had to check if this was not interpreted as `return ({ poolActivities } = user)` (It doesn't). A `;` let's us see the danger by its sheer ugliness, and prevents a bug if the line above changes.
```suggestion
    ;({ poolActivities } = user)
```

Reply:
if it was me i would prevent using this variable definition, and force into `poolActivities = user.poolActivities`
I added `;` in c258dceb4 (exceptionnally not in a fixup, but amended)